src/commands/aem/rde/install.js

```diff
@@ -508,10 +513,19 @@ Object.assign(DeployCommand, {
     }),
     force: Flags.boolean({
       char: 'f',
+      description:
+        'forces the installation, used when the RDE is waiting on an upload from a previous install than can be skipped',
       multiple: false,
       required: false,
     }),
     quiet: commonFlags.quiet,
+    restart: Flags.boolean({
+      char: 'r',
+      description:
+        'restart the environment after a successful installation. This could also be manually done later using the restart command',
```

Comment:
I'd point out that it's not recommended in normal scenarios since restarts are not usually needed and it consumes time (~5 minutes)